### DISCUSSION TITLE
Page interactions conflict with model interactions

### DISCUSSION THREAD

**Discussion by ironandsilk** [google/model-viewer#2010] [Q&A]
### Description

On a long scrollable page, swiping vertically stops the tumble over the X-axis.
You have to press and hold to rotate over the X-Axis when the page is long.

#### Live Demo
Demo recording
[https://www.youtube.com/watch?v=TzgMCPWn35k](url)

based on
https://modelviewer.dev/examples/augmentedreality/index.html#ar

### Version

- model-viewer: vX.X.X

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [X] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
It's a little hard to tell what the expected behavior is from your video; I believe there's a way to record touch-points in the video that might help a bit. Still, I think you might be referring to our `pan-y` default: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-touch-action

Does `touch-action: "none"` have the behavior you expect? Or have I misunderstood?

**Comment by ironandsilk** [comment]
Even here on the top example, when trying to view on top of the item, the hard pull down refreshed the page.
Interactions intended to manipulate the 3D object are conflicting with page and app navigation.
https://www.youtube.com/watch?v=h7w9FgFQuTU&feature=youtu.be

**Comment by elalish** [comment]
Yes, that's expected behavior. This has to do with how much you want to prioritize model interaction over page scrolling. As I said, this can be changed using our `touch-action` attribute; does that give you the behavior you're looking for? We default to allowing the user to scroll the page, because this can get frustrating otherwise, especially if the model covers the whole screen and the page becomes completely unscrollable.
